code/obj/item/device/radios/headsets.dm

```diff
@@ -428,6 +428,18 @@
 	icon_override = "ghost_buster"
 	icon_tooltip = "Ghost Buster"
 
+/obj/item/device/radio/headset/hall_monitor
+	name = "Hall monitor's headset"
+	desc = "So you can listen to(evesdrop on) station security(drama)."
+	icon_state = "sec headset"
+	secure_frequencies = list("g" = R_FREQ_SECURITY, "c" = R_FREQ_CIVILIAN)
+	secure_classes = list(
+		"g" = RADIOCL_SECURITY,
+		"c" = RADIOCL_CIVILIAN,
+		)
+	icon_override = "sec"
+	icon_tooltip = "Security"
```

Comment:
Icon_tooltip is what you see when you hover over the icon in chat, if you don't want them considered real sec you could change this to be something like "hall monitor".